samples/client/petstore-security-test/csharp/SwaggerClient/src/IO.Swagger/Client/JsonSubTypes.cs

```diff
@@ -0,0 +1,281 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace JsonSubTypes
+{
+    //  Copyright 2017 Emmanuel Counasse
```

Comment:
I chose to include the small library as source code instead of an independently licensed package dependency that would increase constraints both on the consumer and producer side. I thought it would not be a problem as I saw that copyright notices exist in other generated files. 

However, I think the copyright notice could be removed as long a little notice explain that it came from 'https://github.com/manuc66/JsonSubTypes'.

What do you think about it?